docs/pages/workflow/expo-cli.md

```diff
@@ -1016,13 +971,104 @@ Alias: `expo ui`
 | `--apple-id [login]` | Apple ID username (please also set the Apple ID password as EXPO_APPLE_PASSWORD environment variable). |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
+| `--config [file]`    | Deprecated: Use app.config.js to switch config files instead.                                          |
 
 </p>
 </details>
 
 <!-- END GENERATED BLOCK. DO NOT MODIFY MANUALLY. -->
+
+## Expo CLI Migration
+
+In Expo SDK 46, we're migrating to a new suite of tooling that is versioned in the `expo` package. Below is a migration guide showing where the legacy `expo-cli` commands live now.
+
+> Legacy `expo-cli` will be in maintenance mode and continue to work while we migrate to the new tooling.
+
+| Legacy               | New                    | Notes                                         |
+| -------------------- | ---------------------- | --------------------------------------------- |
+| `expo init`          | `npx create-expo-app`  | New CLI                                       |
+| `expo start`         | `npx expo start`       | Versioned                                     |
+| `expo export`        | `npx expo export`      | Versioned, `--experimental-bundle` is default |
+| `expo install`       | `npx expo install`     | Versioned                                     |
+| `expo run:android`   | `npx expo run:android` | Versioned                                     |
+| `expo run:ios`       | `npx expo run:ios`     | Versioned                                     |
+| `expo login`         | `npx expo login`       | Versioned                                     |
+| `expo logout`        | `npx expo logout`      | Versioned                                     |
+| `expo register`      | `npx expo register`    | Versioned                                     |
+| `expo whoami`        | `npx expo whoami`      | Versioned                                     |
+| `expo config`        | `npx expo config`      | Versioned                                     |
+| `expo prebuild`      | `npx expo prebuild`    | Versioned                                     |
+| `expo customize:web` | `npx expo customize`   | Versioned, and renamed                        |
+| `expo build:web`     | `npx expo export:web`  | Versioned, and renamed                        |
+| `expo eject`         | `npx expo prebuild`    | Merged into `npx expo prebuild`               |
+| `expo start:web`     | `npx expo start`       | Merged into `npx expo start`                  |
+
+### Deprecated
+
+| Legacy                        | Notes                           |
+| ----------------------------- | ------------------------------- |
+| `expo client:ios`             | Removed in favor of Dev Clients |
+| `expo send`                   | Unimplemented                   |
+| `expo client:install:ios`     | Unimplemented                   |
```

Comment:
we'll probably move the equivalent of this to eas cli, cc @tcdavis

Reply:
This is for installing Expo Go, I might add it as an auxiliary command to the versioned CLI, just don't really want to do that.